README.md

```diff
@@ -76,31 +77,30 @@ transformation code.
 // Toy has literals, variables, basic arithmetic and functions that accept a single argument. 
 
 (define toy_lang
-    (domain
-        (sum expr
-            (lit value::ion)
-            (variable name::symbol)
-            (not expr::expr)
-            (apply op operands::(* expr 2))
-            (call name::symbol argument::expr)
-            (let name::symbol value::expr body::expr)
-            (function var_name::symbol body::expr))
-        (sum op (plus) (minus) (times) (divide))
-))
-
-// Define another type domain which is the same as "toy_lang" but replaces variable names with DeBruijn indices:
-(define toy_lang_nameless
+ (domain
+   (sum operator (plus) (minus) (times) (divide) (modulo))
+    (sum expr
```

Comment:
nit: inconsistent spacing. Also not sure on the PIG convention of indenting 2 or 4 spaces, but it should at least be consistent between `sum operator` and `sum expr`.

Reply:
there isn't an established convention but I do like 4 spaces so I've reformatted this.